### DISCUSSION TITLE
the styles aren't applied correctly Nuxt + tailwindcss 2

### DISCUSSION THREAD

**Discussion by boussadjra** [tailwindlabs/tailwindcss#3478] [Help]
### Describe the problem:
 I've created a project using nuxt.js and I added tailwind by following the steps cited [here](https://tailwindcss.com/docs/guides/nuxtjs#setting-up-tailwind-css) but it doesn't apply the tailwind 2 styles correctly specially the dark mode, sizes and colors.

### Link to a minimal reproduction:
 github repo https://github.com/boussadjra/windy-nuxt
demo https://windy-nuxt.netlify.app/

**Comment by iantearle** [comment]
Have you uncommented dark mode in the tailwind config file? Needs to be media or class

**Comment by boussadjra** [comment]
@iantearle thank you for your reply, I have the darkMode with `class` as value https://github.com/boussadjra/windy-nuxt/blob/main/tailwind.config.js and I've the logic that adds the class dark to the html tag
